src/calculateRentalCost.js

```diff
@@ -1,10 +1,25 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
```

Comment:
Constants must named like shown in video [Transportation on Vacation Solution](https://mate.academy/learn/javascript-advanced/solving-tasks-on-github?section=video&testTaskSlug=js_transportation_on_vacation&learnItemsFilter=All&videoId=4736&theoryId=624)

_CONSTANT_CASE, MACRO_CASE, UPPER_CASE, SCREAM_CASE, SCREAMING_SNAKE_CASE used to declare constant variables_